### ISSUE TITLE
Admin Interface Text Fields Announce Autocomplete Without Autocomplete Functionality

### ISSUE THREAD

**Issue by Santoz2** [ManifoldScholar/manifold#2321]
### Steps to Reproduce ###
1. Navigate to Manifold’s Admin Mode page
2. Navigate to and press the “Add a New Project” link
3. Navigate to the “Title” edit text field
4. Notice how “Title Edit has Autocomplete” is announced
4. Try typing in the text field, or try typing a known name like “The Black Tulip”, and note how there is no indication of Autocomplete upon interaction with the text field

### Observed Behavior ###
Many of Manifold’s text input fields state that they have Autocomplete, but do not properly implement Autocomplete. This can be seen when editing any text field in the Admin Interface (for example, editing the title of a book, or even its year of publication).

### Expected Behavior ###
The user’s ability to interact with an element should be consistent with the information that they are given. If the user is told that the text field they are editing has Autocomplete, it should have Autocomplete. Otherwise, if the field is not intended to have Autocomplete, it should not announce that it has it.

### Test Environment ###
Firefox 60.8.0esr
NVDA 2019.1.1

---
Please let me know if I should provide any clarification! Thank you!

**Comment by dananjohnson** [comment]
Thanks for your feedback, @Santoz2. We don't explicitly enable autocomplete on any of these fields, which makes me suspect that this functionality is being driven by the browser and not by Manifold. (See [this issue](https://github.com/nvaccess/nvda/issues/6663#issuecomment-268689815) in the NVDA repo that seems related). While we could disable autocomplete on each field, I'm not sure it's in our interest to override the browser's behavior here. But let me know if you think I've misdiagnosed the issue!

**Comment by zdavis** [comment]
Closing due to inactivity. Happy to reopen if we get feedback on this.
